### ISSUE TITLE
🐛 The Extension Is not Working

### ISSUE THREAD

**Issue by portrik** [biomejs/biome-zed#151]
### Zed version

Zed 0.204.5 - Installed via Homebrew

### Extension version

0.2.0

### Biome version

2.2.4

### Operating system

- [ ] Windows
- [x] macOS
- [ ] Linux

### Description

Biome seems to have stopped working in Zed during the last week. In Activity Monitor, I can see that `biome` is taking up 200 % of the CPU. But looking into the LSP logs, no communication is happening between Zed and the LSP.  The issue occurring on macOS Tahoe 26.0 on MacBook Pro M4, if it makes any difference. I have not been able to test this behavior on different OS versions.

This behavior is happening across multiple projects - some monorepos, some plain repos. I've prepared a simple reproduction repo where the issue occurs as well.

In Zed, opening the file nor editing it produces any output in the biome LSP logs. But when running `pnpm biome check`, 2 errors and 3 warnings are reported.

### Steps to reproduce

1. Initialize a new project with `pnpm init`
2. Add biome with `pnpm add -E -D @biomejs/biome`
3. Initialize Biome configuration with `pnpm biome init`
4. Create `index.js` file with some code
5. Open the `index.js` and observe no Biome errors, no warnings
6. Running `pnpm biome check` 

### Expected behavior

1. Opening the file with Zed provides feedback from the Biome LSP and allows the code to be formatted

### Does this issue occur when using the CLI directly?

No

### Link to a minimal reproduction

https://github.com/portrik/biome-repro-zed-extension

**Comment by smitbarmase** [comment]
I’m on Zed 0.205.6 and it seems to work for me. I cloned the attached repo, installed the Biome extension, ran `pnpm i`, and opened `index.ts`. Could you share your Zed logs? They might have errors about the Biome language server initializing.

Diagnostics on `index.ts`:
<img width="600" alt="Image" src="https://github.com/user-attachments/assets/6eead60d-d748-4f58-84c7-15b5f4286b2f" />

Logs:
<img width="600" alt="Image" src="https://github.com/user-attachments/assets/3b69510a-7c16-4e82-a490-de53d24dce7a" />

Just to confirm the diagnostics aren’t coming from another language server, here are my settings:

```jsonc
{
  "languages": {
    "TypeScript": {
      "language_servers": ["biome", "!vtsls"],
      "formatter": { "language_server": { "name": "biome" } }
    }
  }
}
```

**Comment by portrik** [comment]
I can confirm that opening the repository again with a new instance of Zed does not produce the issue.

Maybe this is caused by the LSP instead of the extension? When I experienced the bug, I could see `biome` process taking up ~200 % of the CPU in Activity Monitor.

I am not totally sure how to induce the state again. The last few times this has happened to me was while working with a monorepo that had Biome set up. I have no idea how to reproduce the issue now, as any monorepo I try to use is working correctly.

This issue can probably be closed. I can re-report it in the LSP repository if I encounter the issue again.

Thanks for the triage!

**Comment by dyc3** [comment]
Yeah, it was probably from the LSP side. Closing since we don't have a clear repro.
